app.json

```diff
@@ -14,7 +14,7 @@
           "icon": "./assets/images/logo.png",
           "color": "#ffffff",
           "sounds": ["./assets/sounds/juntos-607.mp3"],
-          "mode": "development"
+          "mode": "production"
```

Comment:
What is the mode used for?

Reply:
The mode is different by published or not, so I try to change it and test if it affects notification.
Now the local version run with expo is no error, but I'm not sure how it runs when published.